drivers/SmartThings/matter-rvc/src/init.lua

```diff
@@ -324,6 +343,82 @@ local function rvc_operational_error_attr_handler(driver, device, ib, response)
   end
 end
 
+local function uppercase_to_camelcase(name)
```

Comment:
```suggestion
local function upper_to_camelcase(name)
```
could be shortened like so?